### PR TITLE
Update named values apim configuration

### PR DESCRIPTION
**Pull request by pasqualespica**
The purpose of this PR is to update named values apim property according to [Verificatore](https://pagopa.atlassian.net/wiki/spaces/PAG/pages/433914753/Verificatore+APIM+policy)

### List of changes

Updated 
  - named values apim with correct values ( DEV and UAT env )
  - upd api_operation_policy for UAT activate 

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by corvettafabio — DISMISSED** [review]
named values ok

**Review by infantesimone — DISMISSED** [review]

**Review by menottistefano — CHANGES_REQUESTED** [review]
l'indirizzo corretto per il nodo PROD è 10.79.20.34. necessario verificare con SIA (Alongi) quale sia la url corretta: ?? https://10.79.20.34/webservices/input ??

File: src/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -77,8 +77,8 @@ api_config_enabled = false
 
 # apim x nodo pagmenti
 nodo_pagamenti_enabled = true
-nodo_pagamenti_psp     = "97735020584,97735020583,97735020582,97735020581"
-nodo_pagamenti_ec      = "77777777777,00493410583"
+nodo_pagamenti_psp     = "-"
```
Comment (menottistefano) [inline src/env/prod/terraform.tfvars:80]: lista aggiornata di PROD configurata su APIM SIA :
97249640588,08658331007,05425630968,06874351007,08301100015,02224410023,02224410023,06529501006,00194450219,02113530345,01369030935,07783020725

File: src/env/prod/terraform.tfvars [hunk 2]
```diff
@@ -77,8 +77,8 @@ api_config_enabled = false
...
+nodo_pagamenti_ec      = "-"
```
Comment (menottistefano) [inline src/env/prod/terraform.tfvars:81]: lista aggiornata di PROD configurata su APIM SIA :
00493410583,09633951000,06655971007,00856930102,02478610583,00221940364,00224320366,00246880397,01315320489,00354730392,00357850395,80008270375,00218770394,00226010395,00202300398,81002910396,00360090393,84002010365,,00242920395,80005570561,80015230347,00236340477,92035800488,03428581205,00114510571,00113430573,00184260040,00103110573,00939820726,00109190579,00122520570,82501690018,80001220773,84515520017,03509990788,84002410540,00482510542,00326070166,01350940019,00197530298,00379480031,06396970482,00460900038,82005250285,82002770236,80013960036,83000970018,84002970162,82500110158,00429530546,01199250158,80003370477,00111190575,81001650548,00096090550,95001650167,00451080063,80038190163,00433320033,00449050061,82002270724,00682280284,00448140541,00344700034,81000550673,00450150065,80002860775,83001970017,00121490577,00383120037,00366270031,80023530167,01504430016

**Review by menottistefano — CHANGES_REQUESTED** [review]

**Comment by pasqualespica** [comment]
> l'indirizzo corretto per il nodo PROD è 10.79.20.34. necessario verificare con SIA (Alongi) quale sia la url corretta: ?? https://10.79.20.34/webservices/input ??

right https://pagopa.atlassian.net/wiki/spaces/PAG/pages/438993588/Network+SIA+vs+PagoPA

**Pull request merged**